installer.sh

```diff
@@ -1,10 +1,57 @@
 #!/usr/bin/env bash
+
+systemctl status ModemManager | grep 'active (running)' > /dev/null 2>&1
```

Comment:
> 
> 
> It can be enabled, but not active?
> 
> This new code-block needs to be below the set and function definitions.

```
if systemctl list-unit-files ModemManager.service | grep enabled; then

	sudo systemctl disable ModemManager > /dev/null
        sudo systemctl stop ModemManager > /dev/null
fi
```
Better?

Reply:
I think using the `is-active` command of `systemctl` can be used here, which would remove the need for using grep. Additional set `--quiet` to prevent output when testing that.

There is no need to use `sudo`, as you already have a root check?